### PR TITLE
Pomodor

### PR DESCRIPTION
**Pull request by souhailshiv**
Difficulty level (1-10): [3]
Estimated time spent (hours): [1]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): [yes I learn how to work with timers]
Other feedback (what can we improve?): []

### CODE REVIEW

File: 52/souhailshiv/Pomodor.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                min = min-1
+                sec = 59
+            else:
+                break
+        elif keyboard.is_pressed("s"):
+            exit()
+        else:
+            sec = sec-1
+
+    beepy.beep(sound=5)
```
Comment (bbelderbos) [inline 52/souhailshiv/Pomodor.py:32]: cool, didn't hear (no pun) about this one

File: 52/souhailshiv/Pomodor.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        print("minuts: ", minut, " ", "seconds: ", second)
+        sleep(1)
+        if second == 0:
+            if minut != 0:
+                minut = minut-1
+                second = 59
+            else:
+                break
+
+        elif keyboard.is_pressed("s"):
```
Comment (bbelderbos) [inline 52/souhailshiv/Pomodor.py:49]: How does the end user know?

File: 52/souhailshiv/Pomodor.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+def Break(minut):
+    print("break time")
+    second = 0
+    while True:
+        print("minuts: ", minut, " ", "seconds: ", second)
+        sleep(1)
+        if second == 0:
+            if minut != 0:
```
Comment (bbelderbos) [inline 52/souhailshiv/Pomodor.py:43]: maybe easier to convert to seconds? then you save some math here

File: 52/souhailshiv/Pomodor.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            sec = sec-1
+
+    beepy.beep(sound=5)
+    Break(xy)
+
+
+def Break(minut):
+    print("break time")
+    second = 0
+    while True:
```
Comment (bbelderbos) [inline 52/souhailshiv/Pomodor.py:39]: seems this code is duplicated, abstract in helper function

**Review by bbelderbos — COMMENTED** [review]
Thanks for submitting your code, some feedback

File: 52/souhailshiv/Pomodor.py (hunk 3 above)
Comment (souhailshiv) [reply 52/souhailshiv/Pomodor.py:43]: maybe

File: 52/souhailshiv/Pomodor.py (hunk 2 above)
Comment (souhailshiv) [reply 52/souhailshiv/Pomodor.py:49]: I used beepy library to make a sound when the count down end

File: 52/souhailshiv/Pomodor.py (hunk 4 above)
Comment (souhailshiv) [reply 52/souhailshiv/Pomodor.py:39]: what do you mean

File: 52/souhailshiv/Pomodor.py (hunk 3 above)
Comment (souhailshiv) [reply 52/souhailshiv/Pomodor.py:43]: maybe

Comment (bbelderbos) [reply 52/souhailshiv/Pomodor.py:43]: I would convert to seconds, makes it much easier

File: 52/souhailshiv/Pomodor.py (hunk 2 above)
Comment (bbelderbos) [reply 52/souhailshiv/Pomodor.py:49]: yep cool, didn't know about this library, thanks

File: 52/souhailshiv/Pomodor.py (hunk 4 above)
Comment (bbelderbos) [reply 52/souhailshiv/Pomodor.py:39]: `Break` and `pomodor` have duplicated code which can be (and should be) prevented by abstracting the duplicated code into a separate method - more info: https://refactoring.guru/smells/duplicate-code

**Pull request merged**